foundation/foundation-upgrade/src/test/resources/test-files/v1_7_0/DataLineageMigration/migration/CdiContainerFactory_1_6_0.java

```diff
@@ -0,0 +1,72 @@
+package com.boozallen.cdi;
+
+/*-
+ * #%L
+ * datalineagev160::Pipelines::Example Data Lineage
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.jboss.weld.environment.se.WeldContainer;
+
+import com.boozallen.aiops.core.cdi.CdiContainer;
+import com.boozallen.aiops.core.cdi.CdiContext;
+import com.boozallen.aissemble.alerting.core.cdi.AlertingCdiContext;
+import com.boozallen.aiops.data.lineage.cdi.DataLineageCdiSelector;
```

Comment:
A: Not totally necessary, but I might include other ways you can reference the package name in addition to the imports.  So maybe a fully qualified type for a variable or return type, and/or a string reference.